pkg/arm/data/master-startup.sh

```diff
@@ -571,6 +571,286 @@ spec:
     name: master-cloud-provider
 EOF
 
+cat >/etc/origin/node/pods/ovs.yaml <<'EOF'
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    kubernetes.io/description: |
+      This static pod launches the openvswitch daemon.
+  labels:
+    app: ovs
+    component: network
+    openshift.io/component: network
+    type: infra
+  name: ovs
+  namespace: openshift-sdn
+spec:
+    hostNetwork: true
+    hostPID: true
+    serviceAccountName: sdn
+    containers:
+      - name: openvswitch
+        command:
+        - /bin/bash
+        - -c
+        - |
+          #!/bin/bash
+          set -euo pipefail
+
+          # if another process is listening on the cni-server socket, wait until it exits
+          trap 'kill $(jobs -p); exit 0' TERM
+          retries=0
+          while true; do
+            if /usr/share/openvswitch/scripts/ovs-ctl status &>/dev/null; then
+              echo "warning: Another process is currently managing OVS, waiting 15s ..." 2>&1
+              sleep 15 & wait
+              (( retries += 1 ))
+            else
+              break
+            fi
+            if [[ "${retries}" -gt 40 ]]; then
+              echo "error: Another process is currently managing OVS, exiting" 2>&1
+              exit 1
+            fi
+          done
+
+          # launch OVS
+          function quit {
+              /usr/share/openvswitch/scripts/ovs-ctl stop
+              exit 0
+          }
+          trap quit SIGTERM
+          /usr/share/openvswitch/scripts/ovs-ctl start --system-id=random
+
+          # Restrict the number of pthreads ovs-vswitchd creates to reduce the
+          # amount of RSS it uses on hosts with many cores
+          # https://bugzilla.redhat.com/show_bug.cgi?id=1571379
+          # https://bugzilla.redhat.com/show_bug.cgi?id=1572797
+          if [[ `nproc` -gt 12 ]]; then
+              ovs-vsctl set Open_vSwitch . other_config:n-revalidator-threads=4
+              ovs-vsctl set Open_vSwitch . other_config:n-handler-threads=10
+          fi
+          while true; do sleep 5; done
+        image: '*** GENERATED ***'
```

Comment:
you need to fix up all of the `*** GENERATED ***` parts and remove the corresponding templates from the sync pod.